### ISSUE TITLE
Training behavior changed dramatically

### ISSUE THREAD

**Issue by RK4444** [Unity-Technologies/ml-agents#3204]
Since I upgraded my ML-Agents environment from V0.12 to 0.13 something weird is happening with the training: It is much faster than before, nearly ten times, bit reward is increasing just slowly

1. Upgrade ML-Agents from V0.12 to V0.13
2. Start 3DBalls example
3. Open Tensorboard and watch the Progress

![TrainingBehaviour](https://user-images.githubusercontent.com/37285455/72217652-2e057280-3531-11ea-8efe-c1bc4c7a7527.png)

Please take a look at the reward and the time needed for the training

I am using:

- Windows 10
- ML-Agents version: V0.13.0
- TensorFlow version: 2.0.0
- Environment: 3DBalls
- Unity Version: 2019.3.0f1

I'm not sure if this is a bug. Did some parameters change?

**Comment by ervteng** [comment]
Hi @RK4444, in the most recent `master` branch, we changed the way steps are reported - they are now per-agent rather than per environment. Since there are 12 agents in the 3DBall scene, we'd expect there to be 12 times more steps than before. 

However, this change has _not_ been officially released yet. We recently changed our release process so that `master` is the active development branch. For the latest stable release (0.13), check it out using this command:

`git clone --branch latest_release https://github.com/Unity-Technologies/ml-agents.git`
